frontend/src/lib/tour/driver.css

```
@@ -0,0 +1,175 @@
+.driver-popover {
+  background: #1e293b !important;
+  color: #f1f5f9 !important;
+  border-radius: 12px !important;
+  box-shadow: 0 25px 50px -12px rgba(0, 0, 0, 0.5) !important;
+  max-width: 400px !important;
+  padding: 20px !important;
+  overflow: visible !important;
+}
+.driver-popover * {
+  overflow-wrap: break-word !important;
+  word-break: break-word !important;
```

Stylelint (17.9.0)

[error] Deprecated keyword "break-word" for property "word-break" (declaration-property-value-keyword-no-deprecated)

(declaration-property-value-keyword-no-deprecated)